sda-commons-starter-web/src/main/java/org/sdase/commons/spring/boot/web/client/OidcClientRequestConfiguration.java

```diff
@@ -29,35 +30,44 @@ public OidcClientRequestConfiguration(ApplicationContext applicationContext) {
   @Bean
   @Lazy
   @ConditionalOnProperty(value = "oidc.client.enabled", havingValue = "true")
-  public RequestInterceptor getOidcRequestInterceptor() {
-    return new OidcClientRequestInterceptor(applicationContext);
+  public RequestInterceptor getOidcRequestInterceptor(
+      @Value("${oidc.client.token-pass-through.enabled:true}") boolean tokenPassThroughEnabled) {
```

Comment:
Are you fine with the property name: `oidc.client.token-pass-through.enabled`?
I put `tokenPassThroughEnabled ` instead of `authenticationPassthrough` as we don't pass authentication but only a token. I will rename it for the Dropwizard alignment, nonetheless.

Reply:
I'd agree that the name `authenticationPassthrough` may not be optimal. But since it's a fixed word in both our libraries, we should continue to use it. Compare `AuthenticationPassThroughClientConfiguration`, for example.